server/sonar-ce/src/main/java/org/sonar/ce/taskprocessor/CeWorkerImpl.java

```diff
@@ -183,8 +183,8 @@ private static Profiler startActivityProfiler(CeTask task) {
     return profiler.startInfo("Execute task");
   }
 
-  private static void stopActivityProfiler(Profiler profiler, CeTask task, CeActivityDto.Status status) {
-    addContext(profiler, task);
+  private static void stopActivityProfiler(Profiler profiler, CeActivityDto.Status status) {
+    profiler.addContext("status", status.name());
```

Comment:
when this change was done in 7.X, the log level of the end message was also changed to always be INFO (instead of INFO in case of success, ERROR in case of failure)

Reply:
you're right, commit deleted.